pallets/loan/src/lib.rs

```diff
@@ -86,6 +98,14 @@ pub struct LoanData<Rate, Amount, AssetId> {
 	asset_id: AssetId,
 	status: LoanStatus,
 	loan_type: LoanType<Rate, Amount>,
+
+	// whether the loan written off by admin
+	// if so, we wont update the write off group on this loan further from permission less call
+	admin_written_off: bool,
+	// write off group index in the vec of write off groups
+	// none, the loan is not written off yet
+	// some(index), loan is written off and write off details are found under the given index
+	write_off_index: Option<u32>,
```

Comment:
Is `write_off_index` dependant on `admin_written_off`? As in, can `write_off_index` be `Some(index)` while `admin_writter_off` being `false`?

Reply:
yes it is possible